docs/en/authorization_endpoint.rst

```diff
@@ -286,4 +284,71 @@ Authorization Response example:
 
     http://rp-test.it/oidc/rp/callback/?code=a032faf23d986353019ff8eda96cadce2ea1c368f04bf4c5e1759d559dda1c08056c7c4d4e8058cb002a0c8fa9a920272350aa102548523a8aff4ccdb44cb3fa&state=2Ujz3tbBHWQEL4XPFSJ5ANSjkhd7IlfC&iss=http%3A%2F%2Fop-test%2Foidc%2Fop%2F
 
+Errors
+++++++
 
+In the event of an error, the OP or RP represent the anomaly message 
+as described in the related tables defined by the `Linee Guida UX SPID`_. 
+
+
+.. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - **Claim**
+     - **Descrizione**
+     - **Supportato da**
+   * - **error**
+     - Vedi :ref:`Codici di errori <codici_errore>`
+     - |spid-icon| |cieid-icon|
+   * - **error_description**
+     - Error description.
+     - |spid-icon| |cieid-icon|
+   * - **state**
+     - It MUST be equal to the *status* value included in the *Authentication Request*. The RP MUST verify that it matches the one sent in the *Authentication Request*.
+     - |spid-icon| |cieid-icon|
+
+
+.. _codici_errore:
+
+Codici di errore
+----------------
+
+.. list-table:: 
+   :widths: 20 20 20 20
+   :header-rows: 1
+
+   * - **Error Code**
+     - **Descrizione**
```

Comment:
Description

Reply:
Fixed in https://github.com/italia/spid-cie-oidc-docs/pull/77/commits/19a3b3ea839f2763010e98c7706453b7b38d4036